README.md

```diff
@@ -31,7 +31,7 @@ Current feature list:
 - <sub>[![TabTrader](https://user-images.githubusercontent.com/1294454/66755907-9c3e8880-eea1-11e9-846e-0bff349ceb87.png)](https://tab-trader.com/?utm_source=ccxt)</sub> **[TabTrader](https://tab-trader.com/?utm_source=ccxt)** – trading on all exchanges in one app. Avaliable on **[Android](https://play.google.com/store/apps/details?id=com.tabtrader.android&referrer=utm_source%3Dccxt)** and **[iOS](https://itunes.apple.com/app/apple-store/id1095716562?mt=8)**!
 - <sub>[![Freqtrade](https://user-images.githubusercontent.com/1294454/114340585-8e35fa80-9b60-11eb-860f-4379125e2db6.png)](https://www.freqtrade.io)</sub> **[Freqtrade](https://www.freqtrade.io)** – leading opensource cryptocurrency algorithmic trading software!
 - <sub>[![OctoBot](https://user-images.githubusercontent.com/1294454/132113722-007fc092-7530-4b41-b929-b8ed380b7b2e.png)](https://www.octobot.online)</sub> **[OctoBot](https://www.octobot.online)** – cryptocurrency trading bot with an advanced web interface.
-- <sub>[![TokenBot](https://user-images.githubusercontent.com/1294454/152720975-0522b803-70f0-4f18-a305-3c99b37cd990.png)](https://www.tokenbot.com)</sub> **[TokenBot](https://www.tokenbot.com)** – discover and copy the best algorithmic traders in the world.
+- <sub>[![TokenBot](https://user-images.githubusercontent.com/1294454/152720975-0522b803-70f0-4f18-a305-3c99b37cd990.png)](https://tokenbot.com/?utm_source=github&utm_medium=ccxt&utm_campaign=algodevs)</sub> **[TokenBot](https://tokenbot.com/?utm_source=github&utm_medium=ccxt&utm_campaign=algodevs)** – discover and copy the best algorithmic traders in the world.
```

Comment:
@cryptos81 Your last merged brought a lot of undesired changes to this PR

Reply:
@carlosmiei, I only had added the single graviex.js file in latest commit.

Reply:
@cryptos81 this PR is showing 154 edited files, please remove those edits from this PR.

It's probably due to a merge and a rebase would fix it:

<img width="1134" alt="Screenshot 2022-02-11 at 13 51 17" src="https://user-images.githubusercontent.com/1294454/153579141-f292b7fb-527a-430a-8ac5-5349a264182c.png">

Reply:
@cryptos81 a git-rebase should fix it: https://git-scm.com/book/en/v2/Git-Branching-Rebasing

Reply:
@cryptos81 Most likely you did a merge from an outdated master branch, that's why some many differences

Reply:
@kroitor, Okay thanks.  Let me check and remove these other unwanted files from this PR.

Reply:
@kroitor @carlosmiei, Removed the unwanted files from the PR